caddyhttp/proxy/upstream.go

```diff
@@ -150,7 +150,8 @@ func (u *staticUpstream) From() string {
 
 func (u *staticUpstream) NewHost(host string) (*UpstreamHost, error) {
 	if !strings.HasPrefix(host, "http") &&
-		!strings.HasPrefix(host, "unix:") {
+		!strings.HasPrefix(host, "unix:") &&
+		!strings.HasPrefix(host, "quic:") {
```

Comment:
Is it really necessary to have the user specify this, or can the reverse proxy infer QUIC from the upstream's Alt-Svc headers?